src/com/github/johnthagen/cppcheck/CppcheckInspection.java

```diff
@@ -114,7 +114,7 @@ private static String prependIncludeDir(@NotNull String cppcheckOptions, @NotNul
     // Example output line:
     // [C:\Users\John Hagen\ClionProjects\test\main.cpp:12]: (style) Variable 'a' is not assigned a value.
     // [main.cpp:12] -> [main.cpp:14]: (performance) Variable 'a' is reassigned a value before the old one has been used.
-    Pattern pattern = Pattern.compile("^\\[(.+?):(\\d+)\\](?:\\s+->\\s+\\[.+\\])?:\\s+\\((\\w+)\\)\\s+(.+)");
+    Pattern pattern = Pattern.compile("^\\[(.+?):(\\d+)](?:\\s+->\\s+\\[.+])?:\\s+\\((\\w+)\\)\\s+(.+)");
```

Comment:
Need to ensure that this doesn't break the regex parsing.

Reply:
Can confirm that this does not seem to break the parsing of the regex

Reply:
before:
https://regex101.com/r/GO7EIs/1
after:
https://regex101.com/r/EQPzC6/1